manifests/install.pp

```diff
@@ -4,20 +4,16 @@
 class candlepin::install {
   assert_private()
 
-  $enable_pki_core = $facts['os']['release']['major'] == '8'
-
   if $candlepin::java_package {
     ensure_packages([$candlepin::java_package])
     Package[$candlepin::java_package] -> Package['candlepin']
   }
 
-  if $enable_pki_core {
```

Comment:
I do wonder what will happen with EL9. If we then reintroduce this conditional it's a bit of a waste to go back and forth. Any idea what will happen there?

Reply:
We know that right now there are no modules at all and you can simply run `dnf install tomcatjss`. Wasn't that the package that's in `pki-core` on EL8? If so, I wouldn't expect that to change during the lifecycle of EL9 and thus keep the conditional in our code as preparation for EL9.